src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -184,114 +184,173 @@ internal sealed partial class RegexCharClass
         // Has to be sorted by the first column
         private static readonly string[][] s_propTable =
```

Comment:
@stephentoub Would it make sense to move this table into its own source file and have the tool generate it automatically, so we no longer need to update it manually?

Reply:
That sounds reasonable, yes. Do you mean making RegexCharClass partial, moving this field into that partial definition, and then changing the tool to write out that file? We'd still need to manually run the tool, but we wouldn't then need to patch the output?